### ISSUE TITLE
A suggestion for symbol 501.2 Paved area in multilevel structures, light traffic

### ISSUE THREAD

**Issue by rem-maps** [OpenOrienteering/mapper#1488]
A suggestion for symbol
501.2 Paved area in multilevel structures, light traffic

These white stripes can be used on paved area but also on yellow, rough yellow, semi-open  (and in complex structures - for instance the Barbican in London - no doubt that can be used also on olive green - canopy or building)

see example at Rome Tor Tre Teste - overpass with grass  - map already ISSprOM
![yellow](https://user-images.githubusercontent.com/59602742/72167521-8ae71880-33cb-11ea-85e6-fa51759796c3.png)

I suggest that the symbol in Mapper has to be only white stripes (without brown) and can be superimposed to any surface (brown yellow etc etc).

Colour for stripes :  I think a good choice is  "White for railway"   (because it goes under Black).

To be done before the next stable release that introduces ISSprOM

### Configuration
Mapper Version: master v20191221.17

**Comment by dg0yt** [comment]
Thanks for your feedback.

This is how I read the standard in the first place:
"501.2 Paved area in multilevel structures (A)" is a variation of "501 Paved area (A)", and ISSprOM says "Colour: brown 30% or 50%, white".

Now ISSpOM also says: "(If upper level is covered by other surface, that symbol shall occur with the same pattern)." So it seems you are right that the brown may be replace by yellow or whatever else.

However this cannot really be called "501.2 Paved area in multilevel structures" if a consistent numbering, naming and grouping of symbols is desired. It is rather a companion of "512.2 Underpass or Tunnel". (@krticka?)

"White for railway" won't work well out of the box in our ISSprOM implementation. It covers track symbols (probably a mistake anyway) and also brown for contours...

**Comment by dg0yt** [comment]
Proposal: "512.x Multilevel structure", an area symbol which is white stripes area combined with an  "512.2 Underpass or tunnel" outline. You only need to add "512 Bridge" entries to the lower level, and pay attention to the corners (cf. control circle).
In the end, this means to drop 501.2.

![Multilevel](https://user-images.githubusercontent.com/13567791/72499096-38758400-3831-11ea-917d-b842d910ac7a.png)
[multilevel-demo.zip](https://github.com/OpenOrienteering/mapper/files/4069637/multilevel-demo.zip)

**Comment by mlerjen** [comment]
I think @rem-maps has a point here, that indeed as it is allowed to combine the stripes with a variety of area symbols it makes sense to reduce the symbol 501.2 to the stripes only. Alternatively we would have to exessively create a hatched variant for each  area symbol...

Some points:
1) The combined symbols dotted outline is to rigid. There are many graphical conflicts that should be fixed manually.
![image](https://user-images.githubusercontent.com/15611653/72504032-53e68c00-383d-11ea-9f71-f107cae2f7ed.png) 
Thus we would need a cutable out-line/invisibile parts of the outline functionality.
2) In addition, the pattern in this combined symbol does not seem to be rotable, which it should be.
3) The IOF MC just made a statement, that buildings  shall not be combined with the stripes. 
![image](https://user-images.githubusercontent.com/15611653/72504474-2a7a3000-383e-11ea-90d4-d4e962b18e95.png)

**Comment by dg0yt** [comment]
1) Granted.
2) Bug in Mapper. The pattern was marked as rotatable.
3) Not an issue. I used white over light browns, which is below buildings. But it seems we need to wait for a new color standard anyway.

The key point is: multilevel structure is orthogonal to paved area.

**Comment by dg0yt** [comment]
And the secondary point is: White stripes is tightly coupled to 512.2 and 512.

**Comment by ollesmaps** [comment]
Sidenote: Same logic can be applied to Narrow ride with different runnability in ISOM. 
In my symbol set there is only one "black" ride and the runnability in different colors is applied over it. This ensures you have a smooth ride, not segmented ride if runnability differs. Worth pursuing?

**Comment by mlerjen** [comment]
@ollesmaps : That is best practice :-)
![image](https://user-images.githubusercontent.com/15611653/72508893-c7d96200-3846-11ea-9282-31be56413f23.png)

**Comment by ollesmaps** [comment]
@mlerjen Exactly :+1:

**Comment by krticka** [comment]
Sorry for not replying earlier, busy days.
The symbol was created to help distinguish multilevel structures in urban environment originally for paved area only. Multilevel areas were always problem for organizers and runners and thus should be avoided for course setting/used sparingly and always emphasized with symbols Crossing point or Crossing section. However with our complex urban environment it was very early found, that it can be helpful to combine it also with the other type of surfaces, so sentence was added to the specification. As @dg0yt correctly noted, the naming is now not consistent and I agree, but this is evolution. We have new symbol which will be used under very different situations and thus some what I would call "phase of codification" is needed. We have just recently discussed this again at MC Meeting in Frankfurt and decided that it is OK to combine it with other symbols representing common surfaces except upper surface being a building (521) and canopy (symbol 522). So the current opinion is that it is usually main running level with buildings (canopy) and combining it with multilevel would create more confusion. We need to give time to this solution and than evaluate all the pros and cons. The strategy of MC is to do minor revision of the ISSprOM after two years from edition, so I expect the discussion to multilevel (and possible changes) for this revision.

I agree with @rem-maps @ollesmaps and @mlerjen that one white stripes symbol for use of multilevel in other area symbols is enough. Combined symbols can have limitations for proper graphical implementation.

**Comment by mlerjen** [comment]
@krticka: You might need to revise the meeting minutes. My interpretation of these is, that in case of building on the upper level one should use canopy for the lower level.
![image](https://user-images.githubusercontent.com/15611653/72525832-55c44580-3865-11ea-8936-ffd14471d0a5.png)

**Comment by krticka** [comment]
I opened the discussion in MC about potential correction of the text in 
minutes.

**Comment by rem-maps** [comment]
yes guys - I think it is very clear what we want/need in specifications (then OOM will follow)

and this is very important so @krticka please fix it in the specifications, before the two years revision

the sentence " except for the upper surface being a building (521) as canopy (symbol 522)
is a specific symbol to be used for this configuration" in the MC minutes is not OK

it is possible to have a building (or even a canopy ) in the upper level

if we have too complex situation: don't use it for competitions

so the rule must be:
we map upper level like normal (even if we have canopies or buildings)
and we add white stripes showing lower level

easy and clean

going deeper into the matter in the specifications maybe we should add something like:
striped white should block only areas but not objects and lines (in the examples path is not blocked and I expect the same for a bush, black cross, or a hedge)
 
@mlerjen examples are very good; this is what they mean to me:
![mlerjen_rem](https://user-images.githubusercontent.com/59602742/72606543-10644e80-391f-11ea-9400-fbf2f5aebbd1.jpg)

**Comment by rem-maps** [comment]
> PS: The combined symbol is just a bonus. The white stripes may be added as a separate symbol next to 512.2. But don't lets call it 501.2.

I agree with @mlerjen and @krticka that combined simple is a constraint - better to leave them separated to better flexibility in drawing.
